tests/kallax.go

```diff
@@ -86,6 +86,9 @@ func NewEventsFixtureStore(db *sql.DB) *EventsFixtureStore {
 // required for this operation.
 func (s *EventsFixtureStore) Insert(record *EventsFixture) error {
 
+	if err := record.BeforeInsert(); err != nil {
```

Comment:
Did you reviewed all the generated code?? wooow...
That was pretty good!!! because you spot a serious bug!
It is needed a `return 0, err` here